src/Stripe.net/Entities/BankAccounts/BankAccount.cs

```diff
@@ -73,7 +75,7 @@ public Customer Customer
         /// <summary>
         /// Whether this object is deleted or not.
         /// </summary>
-        [JsonProperty("deleted", NullValueHandling=NullValueHandling.Ignore)]
+        [JsonProperty("deleted", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
cc @ob-stripe just in case we care about this

Reply:
As long as it passes the StyleCop checks, it's fine.